### PR TITLE
Update metadata db schema if table not exists

### PR DESCRIPTION
**Pull request by manitgupta**
If the metadata db exists, it creates new tables if they do not exist, else noop.
If metadata db is not exist, it creates the db and the required new tables.

### CODE REVIEW

File: webv2/helpers/helpers.go [hunk 1]
```diff
@@ -136,10 +148,9 @@ func CheckOrCreateMetadataDb(projectId string, instanceId string) (isExist bool,
 	}
 	if dbExists {
 		isExist = true
-		return
 	}
 
-	err = createDatabase(ctx, uri)
+	err = createDatabase(ctx, uri, isExist)
```
Comment (aksharauke) [inline webv2/helpers/helpers.go:153]: Can we pass dbExists only instead for isExist?

Comment (manitgupta) [reply webv2/helpers/helpers.go:153]: Done

File: webv2/helpers/helpers.go [hunk 2]
```diff
@@ -37,12 +37,54 @@ const (
...
+	) PRIMARY KEY(StateId, StateVersion)`,
+}
+
 func GetSpannerUri(projectId string, instanceId string) string {
 	return fmt.Sprintf("projects/%s/instances/%s/databases/%s", projectId, instanceId, constants.METADATA_DB)
 }
 
 // Creates the schema for the internal metadata database
-func createDatabase(ctx context.Context, uri string) error {
+func createDatabase(ctx context.Context, uri string, isExist bool) error {
```
Comment (shreyakhajanchi) [inline webv2/helpers/helpers.go:87]: nit: better to rename isExist to dbExists

**Review by shreyakhajanchi — APPROVED** [review]

File: webv2/helpers/helpers.go (hunk 2 above)
Comment (manitgupta) [reply webv2/helpers/helpers.go]: Done

**Review by aksharauke — APPROVED** [review]

**Pull request merged**